spec/tsuru_helper.rb

```diff
@@ -0,0 +1,116 @@
+require 'tempfile'
+require 'fileutils'
+require 'command_line_helper'
+
+require 'rspec/retry'
+
+RSpec.configure do |c|
+  c.add_setting :debug_commands, :default => false
+
+  c.add_setting :deploy_env, :default => ENV['DEPLOY_ENV'] || 'ci'
+
+  c.add_setting :target_platform, :default => ENV['TARGET_PLATFORM'] || 'aws'
+
+  # If nil will be populated based on the default platform
+  c.add_setting :target_api_host, :default =>
+    case c.target_platform
+    when 'aws'
+      "#{RSpec.configuration.deploy_env}-api.tsuru.paas.alphagov.co.uk"
+    when 'gce'
+      "#{RSpec.configuration.deploy_env}-api.tsuru2.paas.alphagov.co.uk"
```

Comment:
Perhaps we could pass the domain across as an environment variable from the Makefile the same way that we're doing for TSURU_USER and TSURU_PASS - I would prefer not having the domain names configured in more than one place as it will undoubtedly cause confusion for us (or others) in the future.